docs/docsite/rst/scenario_guides/cloud_guides.rst

```diff
@@ -13,13 +13,7 @@ Guides for using public clouds are moving into collections. We are migrating the
 .. toctree::
    :maxdepth: 1
 
-   guide_alicloud
    guide_cloudstack
    guide_gce
    guide_azure
-   guide_online
-   guide_oracle
-   guide_packet
-   guide_rax
-   guide_scaleway
    guide_vultr
```

Comment:
```suggestion
```

Reply:
I can't find this guide in the repo so can you just remove this vultr reference in this PR?